### ISSUE TITLE
Metadata Order now ignores Author (seems to be a regression)

### ISSUE THREAD

**Issue by brycelelbach** [tabatkins/bikeshed#1826]
I used to put "Author" instead of "Editor" in metadata order in my C++ committee papers:

```
Metadata Order: Author, This Version, Source, Issue Tracking, Project, Audience
```

"Author" being the alias for "Editor" for WG21 papers.

Starting about 1-2 weeks ago, this stopped working: now papers that have metadata order with "Author" instead of "Editor" omit the authors entirely.

https://api.csswg.org/bikeshed/?force=1&url=https://raw.githubusercontent.com/brycelelbach/wg21_p2195_electronic_straw_polls/3a6db67ff9aa627e00729bceabafa3af5178cf34/electronic_straw_polls.bs

In the above example, there's no authors listed. Compare that with this version, which was statically generated from the same commit as above in September:

https://wg21.link/P2195

Changing "Author" to "Editor" fixes this problem; the issue is that there's at least a few dozen papers out there that are using "Author" in the metadata order, so it's a bit of a pain to update.

**Comment by tabatkins** [comment]
Yeah, that was an intentional change by me to fix #1786, unfortunately. I didn't catch this showing up in JF's papers repository, as far as I remember, so sorry for not being able to give you a heads-up about it.

**Comment by jyasskin** [comment]
I think the conclusion here was that this won't be fixed?
